README.md

```diff
@@ -463,6 +463,7 @@ This is a complete list of all of the options that can be passed to `require("ob
       ["x"] = { char = "", hl_group = "ObsidianDone" },
       [">"] = { char = "", hl_group = "ObsidianRightArrow" },
       ["~"] = { char = "󰰱", hl_group = "ObsidianTilde" },
+      ["!"] = { char = "", hl_group = "ObsidianImportant" },
```

Comment:
This HL group also needs to be defined below.

Reply:
HL Group has been updated in readme